spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -273,8 +308,12 @@
               spouse_ssn: "123456789"
             )
           }
+
           it 'fills pdf with LastName FirstName & LastName FirstName' do
             expect(pdf_fields[name_field]).to eq "Lively Blake E & Reynolds Ryan"
+            expect(pdf_fields["Names as shown on Form NJ1040"]).to eq "Lively Blake E & Reynolds Ryan"
+            expect(pdf_fields["Names as shown on Form NJ1040_2"]).to eq "Lively Blake E & Reynolds Ryan"
+            expect(pdf_fields["Names as shown on Form NJ1040_3"]).to eq "Lively Blake E & Reynolds Ryan"
```

Comment:
[dust] In this test and the previous, if you wanted slightly less repetition and prevent copy-paste errors, you could add a local variable like `expected_name = ` instead of copying the expected name in every one.